tutorials/10_DebuggingREMIND.md

```diff
@@ -0,0 +1,169 @@
+Debugging REMIND
+================
+Collaborative effort, coordinated by Oliver Richters, 31 March, 2022
+
+When running and changing REMIND, a multitude of errors may occur. In such a collaborative effort, they need not have been created by you and the REMIND team will help you solve them. So an important part of debugging is to find out whether something your changes to the model created the error, or whether you were unfortunate enough to find errors you are not responsible for.
+
+First, find out the state of your run by executing this in the run directory:
+```bash
+Rscript -e "modelstats::loopRuns('.')"
+```
+PIK cluster users can access all their runs in one go by executing `rs2 -c`.
```

Comment:
maybe add "(regardless where the runs are located)" somewhere

Reply:
PIK cluster users can access information on all their runs in all directories by executing `rs2 -c` (or `rs2 -a` for only active runs).